collectors/timex.plugin/README.md

```diff
@@ -6,13 +6,16 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/timex
 
 # timex.plugin
 
-This plugin monitors the system clock synchronization state on Linux nodes.
+This plugin monitors the system kernel clock synchronization state on Linux nodes.
```

Comment:
```suggestion
This plugin monitors the system kernel clock synchronization state.
```
There's actually another PR that makes timex.plugin available for FreeBSD and MacOS.  :)

Reply:
yup, was fixed while we were discussing the precise wording on the dashboard, good catch :)